Source/SwiftLintFramework/Rules/CommaRule.swift

```diff
@@ -50,23 +51,60 @@ public struct CommaRule: CorrectableRule, ConfigurationProviderRule {
 
     public func correctFile(file: File) -> [Correction] {
         if validateFile(file).isEmpty { return [] }
-        let pattern = "\\s*\\,\\s*([^\\s])"
 
-        let description = self.dynamicType.description
-        var corrections = [Correction]()
-        var contents = file.contents
+        // captures spaces and comma only
+        let pattern =
```

Comment:
why not share the pattern between `validateFile` and `correctFile`? That should help prevent lint/correct mismatches in the future?